scripts/gitlint/zephyr_commit_rules.py

```diff
@@ -98,6 +98,8 @@ def validate(self, title, _commit):
         violation_message = "Commit title does not follow [subsystem]: [subject] (and should not start with literal subsys:)"
         if not pattern.search(title):
             return [RuleViolation(self.id, violation_message, title)]
+        elif title.startswith('treewide'):
+            return [RuleViolation(self.id, violation_message, title)]
```

Comment:
this is appending a fixed expression to what is otherwise a generic check (named like a specific one, but has a configuration https://github.com/zephyrproject-rtos/zephyr/blob/main/.gitlint#L28-L29 that allows extending well beyond "subsystem"), now the check naming and usage are misleading (see the error message)

I think this check should either be in its own gitlint class should be implemented in the existing config, but then the message should still be changed to reflect the new meaning

Reply:
the message is still valid: `Commit title does not follow [subsystem]: [subject]`, treewide is not a subsystem.
Anyways, update to use previous regex.